src/base/datetime/listbox/style.ts

```diff
@@ -10,6 +10,7 @@ kuc-base-datetime-listbox:lang(ja),
 kuc-base-datetime-listbox:lang(ja) * {
   font-family: "メイリオ", "Hiragino Kaku Gothic ProN", Meiryo,
     sans-serif;
+  font-weight: 400;
```

Comment:
kuc-base-datetime-listbox is not only used in calendar but also in time-picker.
So if modify this file, it will affect date-picker, datetime-picker and time-picker.
I suggest you modify file src/base/datetime/calendar/header/style.ts
Add a CSS setting like below.
```
kuc-base-datetime-calendar-header:lang(ja) kuc-base-datetime-listbox * {
  font-weight: 400;
}
```